frontend/src/components/admin/path-connection/pair-item/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import 'assets/scss/utils';
+
+.pair-delete {
+  background-color: $white;
+  color: $gu-purple;
+  border: 0.1em solid $purple-light;
+  border-radius: 50%;
+  width: 1.5em;
+  height: 1.5em;
+  padding: 0.2em;
+  margin-left: 2em;
+  cursor: pointer;
+}
+
+.pair-action-wrapper {
+  position: absolute;
+  right: 0;
+  right: -2em;
+  top: 50%;
+  transform: translateY(-50%);
+  display: none;
```

Comment:
use bootstrap classes for this: 'd-none'

Reply:
it brokes the element